projects/valtimo/components/src/lib/components/right-sidebar/right-sidebar.component.ts

```diff
@@ -136,14 +159,19 @@ export class RightSidebarComponent implements OnInit, OnDestroy {
     private readonly shellService: ShellService,
     private readonly elementRef: ElementRef,
     private readonly configService: ConfigService,
-    private readonly userSettingsService: UserSettingsService
+    private readonly userSettingsService: UserSettingsService,
+    private readonly cdsThemeService: CdsThemeService
   ) {
     this.frontendVersion = VERSIONS?.frontendLibraries;
-    this.isAdmin$.subscribe(isAdmin => {
-      if (this.hideValtimoVersionsForNonAdmins && !isAdmin) {
-        this.showValtimoVersions = false;
-      }
-    });
+    this._subsciptions.add(
+      this.isAdmin$.subscribe(isAdmin => {
+        if (this.hideValtimoVersionsForNonAdmins && !isAdmin) {
+          this.showValtimoVersions = false;
+        }
+      })
+    );
```

Comment:
You can remove this and the showValtimoVersions variable, it's not used anywhere as far as I can see

Reply:
This cannot be removed, it checks asynchronously if user is admin, hence variable name  hideValtimoVersionsForNonAdmins